util/mime-type.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable } from 'angular2/core';
+
+@Injectable()
+export class MimeTypeService {
```

Comment:
General question: what rule-of-thumb should we use to determine when something goes in `/util` vs `/app/shared`?